src/Tools/ExternalAccess/FSharp/Microsoft.CodeAnalysis.ExternalAccess.FSharp.csproj

```diff
@@ -43,47 +43,6 @@
     <Reference Include="System.ComponentModel.Composition" />
   </ItemGroup>
 
-  <ItemGroup>
```

Comment:
are these able to be removed because of the transitive reference build boss change?

Reply:
So the transitive one only impacted project-to-project references; package references were already flowing transitively. That said, I don't think most of this was used _either_ in this project, since all this project is doing is offering wrappers for code in the other project; this was clearly just a copy/paste of the other project.